hack/graph-util.py

```diff
@@ -467,5 +477,6 @@ def get_release_metadata(node):
 
     args = parser.parse_args()
 
+    token = get_token(args)
```

Comment:
nit (and this is super tiny): I like keyword argument like `args=args` to make it harder to accidentally pass arguments in the wrong order.  In the other hand, it seems very unlikely that `get_tokens` ever grows more positional arguments.